docs/gettinginvolved/plugins/faq.rst

```diff
@@ -0,0 +1,93 @@
+.. _build-flocker-driver-faq:
+
+===
+FAQ
+===
+
+Please read through the following frequently asked questions encountered when building and troubleshooting your driver.
+
+.. contents::
+    :local:
+    :backlinks: none
+
+Driver Development
+==================
+
+Is ``dataset_id`` unique for each volume created?
+-------------------------------------------------
+
+Yes.
+
+Is there some way to get the ``dataset_id`` from flocker given the ``blockdevice_id`` specific to our driver?
+-------------------------------------------------------------------------------------------------------------
+
+No.
+
+Does the Flocker node agent cache any state?
+--------------------------------------------
+
+No.
+The only state cached is in the Flocker control agent.
+
+Is there a script to cleanup volumes leftover from running functional tests?
+-----------------------------------------------------------------------------
+
+Yes.
+After each test case, `detach_destroy_volumes <https://github.com/ClusterHQ/flocker/blob/master/flocker/node/agents/test/test_blockdevice.py>`_ is run automatically to cleanup volumes created by the test case.
+This cleanup call is added as part of `get_blockdeviceapi_with_cleanup <https://github.com/ClusterHQ/flocker/blob/master/flocker/node/agents/test/blockdevicefactory.py>`_ .
+
+Please use ``get_blockdeviceapi_with_cleanup`` in your test wrapper.
+
+I get a lot of output in ``journactl`` and it’s very difficult to track what all is happening, is there an easy way to view the logs?
+-------------------------------------------------------------------------------------------------------------------------------------
+
+Eliot-tree is the preferred way, but it currently does not work due to `a known bug <https://github.com/jonathanj/eliottree/issues/28>`_ . 
```

Comment:
I couldn't find any mention of eliottree in the README - could you send me a link.

Its used as Eliot-tree on the python website: https://pypi.python.org/pypi/eliot-tree/15.0.0